front/tests/assets/operationStudies/simulationSettings/electricalProfiles/electricalProfileON.json

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "stationName": "West_station",
+    "stationCh": "BV",
+    "requestedArrival": "11:22:40",
+    "requestedDeparture": "",
+    "stopTime": "",
+    "signalReceptionClosed": false,
+    "margin": {
+      "theoretical": "",
+      "theoreticalS": "0 s",
+      "actual": "0 s",
+      "difference": "0 s"
+    },
+    "calculatedArrival": "11:22:40",
+    "calculatedDeparture": ""
+  },
+  {
+    "stationName": "Mid_West_station",
+    "stationCh": "BV",
+    "requestedArrival": "",
+    "requestedDeparture": "",
+    "stopTime": "",
+    "signalReceptionClosed": false,
+    "margin": {
+      "theoretical": "",
+      "theoreticalS": "0 s",
+      "actual": "0 s",
+      "difference": "0 s"
+    },
+    "calculatedArrival": "11:45:15",
+    "calculatedDeparture": ""
+  },
+  {
+    "stationName": "Mid_East_station",
+    "stationCh": "BV",
+    "requestedArrival": "",
+    "requestedDeparture": "",
+    "stopTime": "124",
+    "signalReceptionClosed": false,
+    "margin": {
+      "theoretical": "",
+      "theoreticalS": "0 s",
+      "actual": "0 s",
+      "difference": "0 s"
```

Comment:
I assume this behavior is accepted since it doesn't affect the results
@axrolld  can you confirm ?
![image](https://github.com/user-attachments/assets/622bc0a5-0d87-47c6-90d7-19b8927fd840)

Reply:
Yep I meant we should ask a PO if it's normal that we display information here even if they don't seem to be very useful. Let's see what @axrolld will say.